### DISCUSSION TITLE
Is there a config to change the platform to set the url into the contents of an album to not use the pound/hashtag sign?

### DISCUSSION THREAD

**Discussion by Drookoo** [LycheeOrg/Lychee#1872] [Q&A] [answered]
For example sometimes when i drop links into social media, the redirected page will drop anything after the `#` symbol.

Say I want to share this link through a social media story: `https://lycheeorg.github.io/demo/#16102925744307/16102927819488`

The social media platform would drop everything after the `#` and send the user to `https://lycheeorg.github.io/demo/` instead.

**Comment by ildyria** [comment, marked as answer]
You can use `lychee.example.com/r/<albumid>` and `lychee.example.com/r/<albumid>/<photoid>` to bypass those restriction from social networks.

At the moment it is the only alternative as the front end is using the # to determine which album to load. In the future it will be slightly different, unfortunately I do not have much time to work on Lychee at the moment. :(
